### DISCUSSION TITLE
Percentage Rarity on award

### DISCUSSION THREAD

**Discussion by ScottFromDerby** [RetroAchievements/RAWeb#747] [Ideas]
Xbox has a clever method of very quickly showing a player how rare an achievement is by simply providing a percentage of how many players have earned it compared to those who have played the game but haven't earned that achievement.
See: https://www.thurrott.com/wp-content/uploads/sites/2/2017/02/xbox-percentage-1024x576.jpg

This is something we could return back to the player in-app when they earn an achievement, replicating this experience on Xbox: http://totalgamingnetwork.com/attachment.php?attachmentid=8586&d=1478298225&stc=1

**Comment by meleu** [comment]
Although I agree that this would be a neat cosmetic I think that in the long run this feature would be a kind of "visual pollution". Reason: achievement set revisions.

Just like what happens with the retro-ratio (aka "white-points"), after achievement set revisions the recently added achievements have a big amount of white-points. If the game is popular, **boom!** :boom: things like this can happen: https://retroachievements.org/achievements/62666

**Comment by Jamiras** [comment]
Implemented at the API level in #2047. Available to clients using rcheevos 11.1.0+.

Rarity is affected by revisions as mentioned above, but everything that affects retro-ratio also also affects rarity. The most prominent of which is attrition. A popular game may have easy late game achievements but if players quit playing before getting there, they appear more rare, and therefore more difficult in the eyes of the retro-ratio algorithm.
